binnacle/test/SpeedWidget_test.dart

```diff
@@ -12,10 +14,25 @@ void main() {
   );
   testWidgets("Testing position stream", 
     (WidgetTester tester) async {
-      expect(true, true);
+      final positionSubject = new BehaviorSubject<Position>();
+      await tester.pumpWidget(SpeedWidget(positionStream: positionSubject.stream));
+      positionSubject.stream.listen((Position pos) {
+        print("Speed:" + pos.speed.toString());
```

Comment:
Do we need the print statement in the test?

Reply:
I want to keep it unless the value is wrong. The test won't say what the value was updated to, if it couldn't find the text. This at least tells you the event happened in terms of  the stream.